app/models/cohort_columns/destination_from_homelessness.rb

```diff
@@ -12,6 +12,7 @@ class DestinationFromHomelessness < ReadOnly
     attribute :title, String, lazy: true, default: ->(model, _attr) { _(model.translation_key) }
 
     def value(cohort_client) # OK
+      # FIXME: this should have a hidden span with the most recent date, if that's possible
```

Comment:
Did you plan to do this, or is this just left for future work?

Reply:
Oh -- looks like you did it in `app/models/grda_warehouse/cohort.rb`, so:
```suggestion
      # This will return a hidden span with the most recent date for sorting as part of the display value
```